src/Catalyst.Core.Modules.Kvm/DeltaExecutor.cs

```diff
@@ -413,10 +355,7 @@ private void DeployCode(ExecutionEnvironment env, TransactionSubstate substate,
             var recipient = entry.TargetContract == null
                 ? GetAccountAddress(entry.Base.ReceiverPublicKey)
                 : new Address(entry.TargetContract);
-            if (entry.IsValidDeploymentEntry)
-            {
-                recipient = Address.OfContract(sender, _stateProvider.GetNonce(sender));
-            }
+            if (entry.IsValidDeploymentEntry) recipient = Address.OfContract(sender, _stateProvider.GetNonce(sender));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4562326)